package.json

```diff
@@ -38,7 +38,7 @@
   "devDependencies": {
     "broccoli-asset-rev": "^2.7.0",
     "ember-ajax": "^3.0.0",
-    "ember-cli": "~3.3.0",
+    "ember-cli": "github:ember-cli/ember-cli#f724919b2d0455899411908531c9179240f5ef41",
```

Comment:
What's the reason for pointing to this sha?

Reply:
MU support is still behind an experiment flag in Ember CLI, so stable versions do not support this AFAIK. The original Ember blueprints also rely on this: https://github.com/emberjs/ember.js/blob/master/package.json#L111

Reply:
Just spoke to @stefanpenner about this. We should not be pointing to SHAs of ember-cli. e-cli 3.4 was a messed up release, and the current 3.5 beta may be messed up too.

Reply:
yes MU support is still behind an experiment flag, because it is subject to change still. I wouldn't recommend relying on this, in anything but experiments until it at-least hits a beta.

Reply:
Ah, I get it. We are simply offering the experiment here as well. I think that is fine.